g3appdriver/src/main/java/org/rmj/g3appdriver/GRider/Database/AppDatabase.java

```diff
@@ -93,9 +93,12 @@
         EDCPCollectionDetail.class,
         EAddressUpdate.class,
         EMobileUpdate.class,
-        EImageInfo.class}, version = 1)
+        EImageInfo.class,
+        EEmployeeInfo.class,
+        EClientInfo.class,
+        ETokenInfo.class}, version = 1)
 public abstract class AppDatabase extends RoomDatabase {
-    private static final String TAG = AppDatabase.class.getSimpleName();
+    private static final String TAG = "GhostRider_DB_Manager";
     private static AppDatabase instance;
 
     public abstract DEmployeeInfo EmployeeDao();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.